ledger/heavy/exporter/pulse_exporter_server.go

```diff
@@ -80,8 +83,12 @@ func (p *PulseServer) Export(getPulses *GetPulses, stream PulseExporter_ExportSe
 	}
 	currentPN := getPulses.PulseNumber
 	for read < getPulses.Count {
-		topPulse := p.jetKeeper.TopSyncPulse()
-		if currentPN >= topPulse {
+		lastPossiblePulseToExport, err := p.pulses.Backwards(ctx, p.jetKeeper.TopSyncPulse(), p.exportDelay)
```

Comment:
This can return `ErrPulseDoesNotExist` if goes too far back (probably shortly after network start).